framework/src/test/java/org/checkerframework/framework/testchecker/h1h2checker/H1H2Checker.java

```diff
@@ -2,4 +2,7 @@
 
 import org.checkerframework.common.basetype.BaseTypeChecker;
 
+import javax.annotation.processing.SupportedOptions;
+
+@SupportedOptions("conservativeInnerClassEnclosingExprCheck")
```

Comment:
Is this necessary? It shouldn't be, as that option is declared in `SourceChecker`.